### DISCUSSION TITLE
Best model for isolate vocals/dialogue for Sound Design in movies/shor films etc??

### DISCUSSION THREAD

**Discussion by Hannamoora** [Anjok07/ultimatevocalremovergui#1536] [General]
Hey fellas! I'm new with this, I saw that you can select the order to get something that you want so, I was wondering what is the best order/combination to remove noisy things like wind, reverb, room, public noise etc, just to isolate the main vocal or main dialogue for use it in Sound Design for movies, short films, documentary, etc.

**Comment by JestAK** [comment]
I use MDX23C-InstVoc-HQ with this settings:
Segment size - 512
Overlap - 16
